Header header
uint32 front_mm
uint32 back_mm
uint8 front_measure_status
uint8 back_measure_status
uint8 front_status
uint8 back_status
